### PR TITLE
Fix ChorusEffect

### PR DESCRIPTION
**Pull request by 9a24f0**

### CODE REVIEW

File: src/palace.pyx [hunk 1]
```diff
@@ -2379,7 +2379,10 @@ cdef class ChorusEffect(BaseEffect):
 
     @phase.setter
     def phase(self, value: int) -> None:
-        self.properties.phase = value
+        if value >= -180 and value <= 180:
+            self.properties.phase = value
+        else:
+            raise ValueError(f'invalid phase: {value}')
```
Comment (McSinyx) [inline src/palace.pyx]:
```suggestion
        if value < -180 or value > 180:
            raise ValueError(f'invalid phase: {value}')
        self.properties.phase = value
```

**Review by McSinyx — CHANGES_REQUESTED** [review]
The same for others accordingly.  Rmb to pull --rebase from master since branch mismatch.

**Comment by McSinyx** [comment]
Gentle ping!

**Comment by McSinyx** [comment]
Dude, is everything OK?

**Comment by Huy-Ngo** [comment]
Something wrong? @9a24f0

**Comment by McSinyx** [comment]
`git pull --rebase origin master` please, too noisy to be reviewed.

**Comment by 9a24f0** [comment]
> `git pull --rebase origin master` please, too noisy to be reviewed.

I did .-.

**Review by McSinyx — APPROVED** [review]
> I did .-.

I've just ran the command and look how clean the diff is now.  You sure you had the `--rebase` flague?  Anyway LGTM, if Travis agree.  This one should be squashed together though.

**Comment by 9a24f0** [comment]
> > I did .-.
> 
> I've just ran the command and look how clean the diff is now. You sure you had the `--rebase` flague? Anyway LGTM, if Travis agree. This one should be squashed together though.

Idk why I got those mess .-. But I clearly did `git pull --rebase origin master`. After that, git keep saying that my current branch is behind so I have to pull from this branch, that's why I even have the `resolve conflicts` commit before :/

**Comment by McSinyx** [comment]
Strange, maybe not being a weeb gaining be the power of getting things done first try lmao.  No worries tho.

**Pull request merged**